README.md

```diff
@@ -1,2 +1,169 @@
-# JavaScript-Capstone
-This is the first time at Microverse that we will build a project in a group. Apart from technical knowledge, we will need to use a few new processes and tools. We have been preparing for that with a few individual tasks in the Microverse program, and it is a time to connect all of them together.
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Javascript Capstone Project <a name="about-project"></a>
+
+The JavaScript capstone project (remember what they are?) is about building my own web application based on an external API. I will select an API that provides data about a topic that I like and then build the webapp around it.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-html">HTML</a></li>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-css">CSS</a></li>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-javascript">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Comply with linters**
+- **DOM events with Javascript**
+- **Javascript events**
+- **Use objects to store and access data**
+- **Use proper ES6 syntax**
+- **Use npm as software packages system**
+- **Use webpack to bundle JavaScript**
+- **Send and receive data from an API**
+- **Use API documentation**
+- **Understand and use JSON**
+- **Make JavaScript code asynchronous**
+- **Use callbacks and promises**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://carlosigreda.github.io/JavaScript-Capstone/)
```

Comment:
- [ ] App is not properly deployed hence throwing a 404 error request. Please ensure your app is properly deployed.

**Live demo not displaying correctly:**
![image](https://user-images.githubusercontent.com/12745474/236656845-0041e7cf-a1f2-47cf-b632-2cc299174fa0.png)

Reply:
Done it @rloterh, Deployment added.

Reply:
> Done it @rloterh, Deployment added.
Please resubmit your project for review once changes are completed!

Reply:
I do have not enough time to finish testing, can I submit all the work that I did please, I was alone in this week without help. I did part that wasn't mine like the comments section, counters, and pop-ups.

Reply:
> * [x]  App is not properly deployed hence throwing a 404 error request. Please ensure your app is properly deployed.
> 
> **Live demo not displaying correctly:** ![image](https://user-images.githubusercontent.com/12745474/236656845-0041e7cf-a1f2-47cf-b632-2cc299174fa0.png)

Nice fixing this.

Reply:
@CarlosIgreda 
For the deadline being so close, kindly proceed to resubmitting your work.

Cheers! 😃

Reply:
Done it @rloterh, I resubmitted my work 🤓